docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -0,0 +1,98 @@
+
+For instance, if you had a domain configuration with a single cluster named "cluster-1" then your Domain YAML file may have content like this:
+
+```
+spec:
+  ...
+  clusters:
+    - clusterName: cluster-1
+      replicas: 3
+  ...
+```
+
+If you modified your WebLogic domain configuration (using the console or WLST) to add a new dynamic cluster named "cluster-2", then you could immediately start cluster members of this new cluster by updating your Domain YAML file like this:
+
+```
+spec:
+  ...
+  clusters:
+    - clusterName: cluster-1
+      replicas: 3
+    - clusterName: cluster-2
+      replicas: 2
+  introspectVersion: "2"
+  ...
+```
+
+When this updated Domain YAML file is applied, the operator will initiate a new introspection of the domain configuration during which it will learn about the additional WebLogic cluster and then the operator will continue to start WebLogic Server instances that are members of this new cluster. In this case, the operator will start two Managed Servers that are members of the cluster named "cluster-2".
+
+#### Distributing changes to configuration overrides
+
+The operator supports customer-provided [configuration overrides]({{<relref "/userguide/managing-domains/configoverrides/_index.md">}}). These configuration overrides, which are supported with Domain in PV or Domain in Image, allow you to override elements of the domain configuration, such as data source URL's or credentials.
+
+With operator 3.0.0, you can now change the configuration overrides and distribute these new configuration overrides to already running WebLogic Server instances. To do this, update the ConfigMap that contains the configuration overrides or update one or more of the Secrets referenced by those configuration overrides and then initiate introspection by changing the `introspectVersion` field.
+
+We have introduced a new field, called `overrideDistributionStrategy` and located under `configuration`, that controls whether updated configuration overrides are distributed dynamically to already running WebLogic Server instances or if the new configuration overrides are only applied when servers are started or restarted.
+
+The default value for `overrideDistributionStrategy` is DYNAMIC, which means that new configuration overrides are distributed dynamically to already running WebLogic Server instances. 
+
+Alternately, you can set `overrideDistributionStrategy` to ON_RESTART, which means that the new configuration overrides will not be distributed to already running WebLogic Server instances, but will instead be applied only to servers as they start or restart. Use of this value will *not* cause WebLogic Server instances to restart absent changes to other fields, such as `restartVersion`.
```

Comment:
It'd be helpful to have a bolded and/or top level section that discusses dynamic override limitations.  Maybe this is a good candidate for 'notice' box.  

Specifically, we should call out that dynamic overrides on running WL servers can only take effect if the corresponding mbean attribute is 'dynamic' (for example datasource Url and username fields are _not_ dynamic, while the encryptedPassword field is).  Perhaps include a similar notice in the config-overrides doc.